### DISCUSSION TITLE
Download all podcast episodes?

### DISCUSSION THREAD

**Discussion by CaptSneeze** [zotify-dev/zotify#9] [Q&A]
Apologies, I am trying to sort this out myself but was unable to determine a way to do so. I can download individual podcast episodes, using the URL for each episode. I can't figure out a way to download all episodes for a given podcast on Spotify. Am I doing something incorrectly, or is this not possible?

**Comment by ITReconnecting** [comment]
> Apologies, I am trying to sort this out myself but was unable to determine a way to do so. I can download individual podcast episodes, using the URL for each episode. I can't figure out a way to download all episodes for a given podcast on Spotify. Am I doing something incorrectly, or is this not possible?

Hi, were you able to work out how to do this in the end? I also want to download every episode from a podcast and cannot work it out so far,

**Reply by CaptSneeze** [reply]
I never figured out a clean way of doing it with Zotify. I eventually reached a point where I was spending much more time trying to script it, and just brute forced it in the end. I manually captured all of the required URLs (which ended up also taking a **long** time), then I scripted the second part where zotify downloads one at a time.
